README.md

```diff
@@ -243,18 +238,6 @@ The repository is prepared to deploy the service using GitHub actions. This is t
 
    <p align="center"><img src="docs/images/deploy-service-workflow.png" alt="Deploy service workflow" width=80%></p>
 
-   You should see the following output in the *Summary* step of the workflow:
```

Comment:
Does really removing this helps? How will the user know the IP address to connect otherwise? It's not explained anywhere else in the repo.